ResearchKit/Common/ORKScaleSlider.m

```diff
@@ -96,10 +164,9 @@ - (void)updateValueForTouchAtPoint:(CGPoint)touchPoint {
 }
 
 static CGFloat kLineWidth = 1.0;
-- (void)drawRect:(CGRect)rect
-{
-    CGRect trackRect = [self trackRectForBounds:[self bounds]];
-    CGFloat centerY = [self bounds].size.height / 2.0;
+- (void)drawRect:(CGRect)rect {
+    CGRect trackRect = [self trackRectForBounds:self.bounds];
+    CGFloat centerY = self.bounds.size.height / 2.0;
```

Comment:
It is a property, but ObjC property accesses are still method calls, and they're still relatively expensive compared to accessing (for example) a member of a struct.